database/migrations/2021_01_27_085903_config_map_display_direction.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/** @noinspection PhpUndefinedClassInspection */
+
+use App\Models\Configs;
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Support\Facades\DB;
+
+class ConfigMapDisplayDirection extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
+		defined('BOOL') or define('BOOL', '0|1');
+
+		DB::table('configs')->insert([
+			[
+				'key' => 'map_display_direction',
+				'value' => '1',
+				'confidentiality' => 1,
```

Comment:
I'm not sure if it is a confidentiality `1` or a `0`. It might actually be a `0` because maps may be accessible while not being logged in.

Reply:
I wasn't 100% clear on the confidentiality values — are they documented someplace? Happy to make this whatever you think it should be, obviously. Will push a change to make it 0.

Reply:
Not really documented aside from in the code...
https://github.com/LycheeOrg/Lychee/blob/master/app/Models/Configs.php#L244
https://github.com/LycheeOrg/Lychee/blob/master/app/ModelFunctions/ConfigFunctions.php